backend/test/fixtures/database/__init__.py

```diff
@@ -4,9 +4,9 @@
 
 from sqlalchemy import func
 
-from server.db.cellxgene_orm import CellxGeneUser, CellxGeneDataset, Annotation, Base
-from server.db.create_db import create_db
-from server.db.db_utils import DbUtils
+from backend.czi_hosted.db.cellxgene_orm import CellxGeneUser, CellxGeneDataset, Annotation, Base
```

Comment:
shouldn't this file be in `backend/czi_hosted/test/fixtures/...`?

Reply:
Moved all of the tests/infrastructure into test directory